OIDC add display_errors param (#2227)
Kong/docs.konghq.com: app/_hub/kong-inc/openid-connect/index.md

`audience_claim`                        | The Name of the claim (or a path) where the audience can be found.
`domains`                               | The domains to be verified against the `hd` claim.
`max_age`                               | The `max_age` (in seconds) for the previous authentication, specifically the `auth_time` claim.

> `display_errors`                        | Display additional debugging information to the user when there's an error. Not recommended to be enabled in a production environmment.

Reviewer: ```suggestion
`display_errors`                        | Display additional debugging information to the user when there's an error. Not recommended to be enabled in a production environment.
```

* `86400`
* `1800`

#### config.display_errors

> By enabling this setting, consumers will be shown additional debugging information if they try and log on but run into an error. This setting should be treated with care and preferrably should not be enabled in Production environments.

Reviewer: ```suggestion
You can enable this setting to show consumers additional debugging information if they try to log on but run into an error. 
> **Warning:** This setting should be treated with care. In most cases, it should not be enabled in Production environments.
```
